projects/ngx-extension/src/lib/operators/start-with-tap.operator.spec.ts

```diff
@@ -17,13 +17,24 @@ describe('startWithTap', () => {
   });
 
   it('works with an observable that emits multiple values', (done) => {
-    const callbackSpy = jasmine.createSpy('callback');
-    const source$ = of(1, 2, 3);
+    let callbackTime: number;
+
+    const callbackSpy = jasmine.createSpy('callback').and.callFake(() => (callbackTime = Date.now() - startTime));
+
+    const count = 4;
+    const period = 100;
+    const source$ = interval(period).pipe(take(count));
 
+    const startTime = Date.now().valueOf();
```

Comment:
1. Is valueOf() not necessary? `Date.now()` works well I guess.
2. Can we move this line to line 27 and have a line between `const startTime` and `source$.pipe`